### DISCUSSION TITLE
Retrieving Store Social Links

### DISCUSSION THREAD

**Discussion by martonmegyeri** [Shopify/storefront-api-feedback#204] [Ideas & Requests]
Hi,
Currently everything from the brand settings (logo, colors, slogan, etc.) are available on the Storefront API but the social links.
By introducing this feature, it would make it possible to dynamically display corresponding social links within different sections of a Hydrogen store, such as the footer. 

<img width="1728" alt="image" src="https://github.com/Shopify/storefront-api-feedback/assets/35447902/6b13813f-89f1-47f1-a532-076e399238c5">

**Comment by navneetkumar-pim-webkul** [comment]
You only can get the following things
brand colors, a short description, a slogan, and a cover image.
social links are not available in storefront API. Social Links you can manage using the shop meta fields.

```
{
  shop {
    brand {
      colors {
        primary {
          background
          foreground
        }
        secondary {
          background
          foreground
        }
      }
      shortDescription
      slogan
      coverImage {
        id
        image {
          altText
          url
        }
        alt
      }
      logo {
        image {
          url
        }
      }
    }
  }
}

**Reply by martonmegyeri** [reply]
Thanks for your reply, I've changed the category of the discussion to feature request.
